proto/warden/async/v1beta1/task.proto

```
@@ -15,6 +15,10 @@
 
 package warden.async.v1beta1;
 
+import "cosmos/base/v1beta1/coin.proto";
```

Buf (1.47.2)

import "cosmos/base/v1beta1/coin.proto": file does not exist

(COMPILE)